obal/data/roles/repoclosure/tasks/dist.yml

```diff
@@ -0,0 +1,18 @@
+- name: 'Run repoclosure for {{ item }}'
```

Comment:
Should we use something other than `item` thats a bit more clear? and then use a `loop_var` to set this on the task using this?

Reply:
sure can, good idea